release-team/role-handbooks/release-notes/README.md

```diff
@@ -214,11 +214,13 @@ Begin running release-notes tool for the ongoing collection of release notes wit
 - __Release day__
 - Copy notes from Google Doc to HackMD in markdown
 - Final version of release notes committed for release
+- Close the _Known Issues_ Issue is and make sure everything has been resolved
 - Release Notes must be merged into master prior to the release. If this is not done the release will include the latest draft.
 
 ### Week 19
 
 - Release retrospective participation
+- Update the TODOs section at the end of each release for the next Release Notes Team
```

Comment:
nice!

Reply:
Also would want to update other things like tool documentation, debugging tips, etc